swiss/src/app/landsraad/landsraad.component.ts

```diff
@@ -103,4 +155,10 @@ export class LandsraadComponent implements OnInit {
     this.resultsShown = event.checked
     this.db.object("landsraad/votingConfig/resultsShown").set(event.checked)
   }
+
+  questionEligibleToDisplay(question: ValueNameExtended, showHiddenQuestions: boolean, filterByRoundId: string) {
+    return question["decretType"] !== "- Žádný -"
```

Comment:
This will of course change to `!question["decretType"]` after I refactor the 1st one.